File: ifElse.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int age;
    cin>>age;
    string result;
    if(age>=18) result = "Adult";
    if(age<18 && age>10) result = "Teen";
    else result = "Child";
    cout<<result<<endl;
}

File: ifElse.java
import java.util.Scanner;
class ifElse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int age = scanner.nextInt();
        if(age>=18) {
            System.out.print("Adult");
        } else {
            System.out.print("Teen");
        }
        scanner.close();
    }
}